mllib/src/test/scala/org/apache/spark/mllib/tree/GradientBoostedTreesSuite.scala

```diff
@@ -111,8 +115,10 @@ class GradientBoostedTreesSuite extends SparkFunSuite with MLlibTestSparkContext
           EnsembleTestHelper.validateClassifier(gbt, GradientBoostedTreesSuite.data, 0.9)
         } catch {
           case e: java.lang.AssertionError =>
+            // scalastyle:off println
             println(s"FAILED for numIterations=$numIterations, learningRate=$learningRate," +
```

Comment:
It looks like logs don't show at all for tests. If you want to print something from inside a test `println` and `throw exception` seem to be the way to go.